src/platform/silabs/tracing/BUILD.gn

```diff
@@ -18,28 +18,33 @@
 import("//build_overrides/build.gni")
 import("//build_overrides/chip.gni")
 import("${chip_root}/build/chip/buildconfig_header.gni")
-
-declare_args() {
-  # When enabled, this sets:
-  #  - configGENERATE_RUN_TIME_STATS in FreeRTOS
-  #  - configUSE_STATS_FORMATTING_FUNCTIONS in FreeRTOS
-  #  - Runtime statistics timer configuration
-  #  - Task switching hooks for statistics collection
-  silabs_tracing_runtime_stats = false
-}
+import("${chip_root}/src/platform/silabs/tracing/args.gni")
 
 buildconfig_header("silabs_tracing_config") {
   header = "SilabsTracingConfig.h"
   header_dir = "platform/silabs/tracing"
 
   defines = []
 
+  if (silabs_tracing_energy_stats == true) {
+    defines += [ "SILABS_TRACING_ENERGY_STATS=1" ]
+  }
+
   if (silabs_tracing_runtime_stats == true) {
     defines += [
       "TRACING_RUNTIME_STATS=1",
       "TRACING_RUNTIME_STATS_MAX_TASKS=16",
     ]
   }
+
+  if (silabs_tracing_energy_traces) {
+    defines += [
+      "SILABS_TRACING_ENERGY_TRACES=1",
```

Comment:
Duplicated from line 29 to 32. Well it think here is better and we should remove the line 29-32

also a small NIT: Our specific defines are typically `SL_` or `SL_MATTER_` rather than SILABS_